src/types/PerformanceTable.cpp

```diff
@@ -18,7 +22,9 @@ PerformanceTable::PerformanceTable(std::vector<Performance> &perf_vect) {
     // ensure there is no performance with dupplicated name
     if (std::find(perf_id_vect.begin(), perf_id_vect.end(), p.getId()) !=
         perf_id_vect.end()) {
-      throw std::invalid_argument("Each performance must have different ids.");
```

Comment:
What is the issue with this check here? It's an important assumption

Reply:
I took out